crates/moon/src/frontend.rs

```diff
@@ -1,8 +1,10 @@
 use crate::CONFIG;
+use lang::Lang;
 use std::borrow::Cow;
 use tokio::fs;
 
 pub struct Frontend {
+    pub(crate) lang: Lang<'static>,
```

Comment:
I would guess the compiler suggested to add `'static` here but that's not what we want.

When you write `Lang<'static>` then `Lang`'s variant `Custom` becomes `Lang::Custom<'static str>`. 
The compiler is happy but a future MZ app developer won't be so much happy when the dev realizes it's possible to pass only static str (e.g. `"something"`) to the `Custom` variant. It means it won't be possible to load the lang from a database or receive from the fronted and pass the value as `String` to the `Custom` variant. 

We have some other options here:
1) `Frontend<'a> { pub(crate) lang: Lang<'a> ...` ; I'm not sure how much `Frontend`s `'a` would be "contagious" - there is a chance it would be annoying or impossible to use in practice. You would need to try it, I'm not brave enough to guess the outcome.
2) Remove the lifetime from the `Lang` enum altogether and replace `Custom<&'str>` with `Custom<String>` or perhaps better alternative `Custom<Cow<'static, str>>`. `Cow` is basically an `enum` with two variants - `Owned` (aka `String` and `Borrowed` (aka `&'static str`) in our case. It's theoretically a little bit slower then just plain `&'str` but still a much better option then to have allocate unnecessary `String` memory even for passed `&'static str`. 

Hope it makes sense, strings are quite complicated in Rust.